rpc/core/routes.go

```diff
@@ -47,16 +47,18 @@ func (env *Environment) GetRoutes() RoutesMap {
 		"broadcast_tx_async":  rpc.NewRPCFunc(env.BroadcastTxAsync, "tx"),
 
 		// abci API
-		"abci_query": rpc.NewRPCFunc(env.ABCIQuery, "path,data,height,prove"),
-		"abci_info":  rpc.NewRPCFunc(env.ABCIInfo, "", rpc.Cacheable()),
+		"abci_query":                  rpc.NewRPCFunc(env.ABCIQuery, "path,data,height,prove"),
+		"abci_query_with_attestation": rpc.NewRPCFunc(env.ABCIQueryWithAttestation, "path,data,height"),
+		"attestor_pub_key":            rpc.NewRPCFunc(env.AttestorPubKey, ""),
```

Comment:
good to move to below and make attestor API section?

```suggestion
        // attestor API
		"attestor_pub_key":            rpc.NewRPCFunc(env.AttestorPubKey, ""),
```